src/main/java/frc/robot/commands/autonomous/TestAutoCommands.java

```diff
@@ -53,26 +53,47 @@ public TestAutoCommands(
     this.clawOpenClose = clawOP;
   }
 
-  public SequentialCommandGroup starting(String path, double maxVel, double maxAcc) {
-    return new SequentialCommandGroup(
-        new ParallelCommandGroup(
-            new ClawEncoderMoveDown(-100, clawPivot, clawEncoder, "Cube").withTimeout(1.5),
-            new PPSwerveFollower(
-                driveSystem, poseEstimatorSystem, "move12", new PathConstraints(.5, .5), true)),
-        new MagicMotionPos(mainArm, 40, 1, 1, .5),
-        new MagicMotionPos(mainArm, 190, 2.75, 5, .5),
-        new MagicMotionPos(mainArm, 10, 3, 1.5, .5),
-        Commands.waitSeconds(.25),
-        new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5),
-        new ParallelCommandGroup(
-            new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5),
-            new ClawEncoderMoveUp(0, clawPivot, clawEncoder, "Cube"),
-            new PPSwerveFollower(
-                driveSystem,
-                poseEstimatorSystem,
-                path,
-                new PathConstraints(maxVel, maxAcc),
-                true)));
+  public SequentialCommandGroup starting(String path, double maxVel, double maxAcc, String piece) {
```

Comment:
helpful to have this code in one place, so we can have the cone/cube choice ready